packages/veritone-react-common/src/components/DataTable/table.test.js

```diff
@@ -151,6 +151,25 @@ describe('MenuColumn', function() {
         .length
     ).toEqual(0);
   });
+
+  it('excludes specified actions', function() {
+    let data = {
+      actions: ['delete', 'alter', 'manage']
+    };
+    let wrapper = mount(
+      <SupressColumnWarnings>
+        <MenuColumn
+          data={data}
+          dataKey="actions"
+          protectedActions={['delete']}
+          excludeActions={['alter']}
+        />
+      </SupressColumnWarnings>
+    );
+
+    const menuItems = wrapper.find('Menu').prop('children');
+    expect(map(menuItems, 'key')).not.toContain('alter');
```

Comment:
lets put an affirmative test too -- verify that delete and manage show up.

Reply:
added